articles/rules/_includes/_context-prop-authentication.md

```diff
@@ -0,0 +1,26 @@
+An object containing information related to the authentication transaction with the following properties:
+
+- `methods`: an array of objects containing the authentication methods a user has completed during their session. For example, a user that has completed a password-based authentication followed by MFA may have the following methods:
+
+```json
+[
+  {
+    "name": "pwd",
+    "timestamp": 1434454643024
+  },
+  {
+    "name": "mfa",
+    "timestamp": 1534454643881
+  }
+]
+```
+
+The method objects will contain the following properties:
+
+- `name`: a string representing the name of authentication method that has been completed. It can be one of the following values:
```

Comment:
Could we clarify that more values could be supported in the future (eg: `non exhaustive list`)?

Reply:
Added clarification 👌